main/webapp/modules/core/styles/views/data-table-view.css

```diff
@@ -146,14 +163,13 @@ table.data-table td.column-header, table.data-table th.column-header {
 }
 
 .column-header-name {
-  margin: 0 0 0 21px;
   padding: 4px 0 0 0;
-  display: block;
+  display: inline-block;
+  position: absolute;
 }
 
 a.column-header-menu {
-  float: left;
-  display: block;
+  display: inline-block;
```

Comment:
I cleaned up this part to help with the resizing. I am not sure why the column menu was a `float: left` which was intended to lodge itself in the margin of the column header. It feels cleaner with two inline elements next to each other, but there might be ways to improve this even more.